docs/integrations/airflow.md

```diff
@@ -52,6 +54,35 @@ SQLMesh supports a variety of engines in Airflow. Support for each engine is pro
 
 * [BigQuery](engines/bigquery.md#airflow-scheduler)
 * [Databricks](engines/databricks.md#airflow-scheduler)
+* [Postgres](engines/postgres.md#airflow-scheduler)
 * [Redshift](engines/redshift.md#airflow-scheduler)
 * [Snowflake](engines/snowflake.md#airflow-scheduler)
 * [Spark](engines/spark.md#airflow-scheduler)
+
+## Managed Airflow instances
+
+Multiple companies offer managed Airflow instances that integrate with their products. This section describes SQLMesh support for some of the options.
+
+### Google Cloud Composer
+
+SQLMesh fully supports Airflow hosted on [Google Cloud Composer](https://cloud.google.com/composer/docs/composer-2/composer-overview) - see the [configuration reference page](../reference/configuration.md#cloud-composer) for more information.
+
+### Astronomer
+
+Astronomer provides [managed Airflow instances](https://www.astronomer.io/product/) running on AWS, GCP, and Azure. SQLMesh fully supports Airflow hosted by Astronomer.
+
+### AWS MWAA
+
+SQLMesh does not fully support AWS MWAA (Amazon Managed Workflows for Apache Airflow) because MWAA does not expose the standard Airflow API. However, SQLMesh can run scheduled DAGs on MWAA via an external state connection.
+
+In this approach, `sqlmesh plan` and `sqlmesh run` behave differently:
+
+- The `sqlmesh plan` command is issued by users. When they issue the command and choose to backfill, the built-in scheduler executes the project's models in the data warehouse.
+- The `sqlmesh run` command is issued by the MWAA Airflow instance on an appropriate cadence - users do not run the command themselves. MWAA then runs the DAG and executes the models in the data warehouse.
```

Comment:
We should also mention that the Janitor process will also run automatically on Airflow with this setup